.travis/build.sh

```diff
@@ -38,44 +38,61 @@ mvn spotbugs:check
 # Run testsuite with java 8 only
 if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
 
-  docker pull oryd/hydra:v1.8.5
+  arch=$(uname -i)
```

Comment:
`uname -i` doesn't work on MacOS, breaking the build.

```
uname: illegal option -- i
usage: uname [-amnprsv]
```

Reply:
Changed to `uname -m` - this should work.

Reply:
Looks good to me.